infra/integration-image/Dockerfile.test

```diff
@@ -57,3 +57,31 @@ RUN curl -fsSLO https://releases.hashicorp.com/packer/0.12.2/packer_0.12.2_linux
 
 RUN curl -sL https://deb.nodesource.com/setup_7.x | bash -
 RUN apt-get install -y nodejs
+
+# Add docker in docker support
```

Comment:
isn't that the next step after this PR is merged(modifications on this Dockerfile are approved)?